.env.example

```
@@ -1,6 +1,45 @@
+NEO4J_PASSWORD=change-this-password
+# Absolute or relative path to your Obsidian vault directory
+OBSIDIAN_VAULT_PATH=./vault
+
+# Email settings (optional; only required for lifecycle email notifications)
+# If you want to receive lifecycle reports via email, fill in the following:
+SMTP_HOST=smtp.gmail.com
+SMTP_PORT=587
+SMTP_USER=username@gmail.com
+EMAIL_TO=username@gmail.com
+# LINEAR_API_KEY is required for Linear integration (see integration points in documentation)
+LINEAR_API_KEY=your-linear-api-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LINEAR_API_KEY key should go before the SMTP_HOST key

(UnorderedKey)